usaspending_api/accounts/urls.py

```diff
@@ -1,27 +1,11 @@
 from django.conf.urls import url
```

Comment:
Similar question that I had about common.py in `views/` (below). Since this File C data is the only thing here and is keyed by tas, what do you think about moving these urls to `tas.py` and getting rid of `common.py` since it's not really "common" as much as it is "orphaned."